src/mergeQueue.ts

```diff
@@ -10,19 +10,22 @@ export const run = async () => {
   const pendingMerge = await fetchPendingMerge();
 
   // group PRs by milestone
-  const milestoneToPr = new Map<string, number[]>();
+  const milestoneToPr = new Map<
+    string,
+    { number: number; user: { login: string } }[]
```

Comment:
What about `name` instead of `login`?
I had to look below to find out that you mean the name, and not some type of secret or authentication method…

Reply:
Ah wait, does it come like that from the GitHub API?
If yes, ignore my comment.

Reply:
If no, then why do you not assign the user manually?

Reply:
it's from the GitHub API